coverage/base.css

```diff
@@ -0,0 +1,224 @@
+body, html {
```

Comment:
Hi Adalid, it looks like `coverage/` folder was accidentally added. Contents of this folder must not be tracked, please remove it